tools/cli/go.mod

```diff
@@ -19,6 +19,8 @@ require (
 	golang.org/x/net v0.0.0-20220114011407-0dd24b26b47d
 	golang.org/x/oauth2 v0.0.0-20211005180243-6b3c2da341f1
 	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.23.4
+	k8s.io/apimachinery v0.23.4
```

Comment:
nit: few lines below, these get replaced
```
	k8s.io/api => k8s.io/api v0.19.12
	k8s.io/apimachinery => k8s.io/apimachinery v0.20.0			k8s.io/apimachinery => k8s.io/apimachinery v0.20.0
```
Any chance we can remove those replaces to depend on newer k8s modules?

Reply:
I was unable to make it work without `replace` but I have updated replace entries to the newest version.